mapreduce/src/main/java/org/pentaho/hadoop/mapreduce/PentahoMapReduceBase.java

```diff
@@ -168,9 +172,10 @@ public void configure( JobConf job ) {
     }
 
     if ( debug ) {
```

Comment:
This is not how it should be. This should be:

```
if (log.isDebugEnabled()) {
  log.logDebug( message );
}
```

Do not use "if (debug)" because this is hardcoded and will never be enabled. 

Do not use "setLogLevel()" because you're overriding the system administrator's configuration.